github/copilot.go

```diff
@@ -63,6 +65,39 @@ type SeatCancellations struct {
 	SeatsCancelled int `json:"seats_cancelled"`
 }
 
+// CopilotUsageSummaryListOptions represents the optional parameters to the CopilotService.GetOrganizationUsage method.
+type CopilotUsageSummaryListOptions struct {
+	Since *time.Time `url:"since,omitempty"`
+	Until *time.Time `url:"until,omitempty"`
+
+	ListOptions
+}
+
+// CopilotUsageBreakdown represents the breakdown of Copilot usage for a specific language and editor.
+type CopilotUsageBreakdown struct {
+	Language         string `json:"language"`
+	Editor           string `json:"editor"`
+	SuggestionsCount int    `json:"suggestions_count"`
+	AcceptancesCount int    `json:"acceptances_count"`
+	LinesSuggested   int    `json:"lines_suggested"`
+	LinesAccepted    int    `json:"lines_accepted"`
+	ActiveUsers      int    `json:"active_users"`
+}
+
+// CopilotUsageSummary represents the daily breakdown of aggregated usage metrics for Copilot completions and Copilot Chat in the IDE across an organization.
+type CopilotUsageSummary struct {
+	Day                   string                   `json:"day"`
+	TotalSuggestionsCount int                      `json:"total_suggestions_count"`
+	TotalAcceptancesCount int                      `json:"total_acceptances_count"`
+	TotalLinesSuggested   int                      `json:"total_lines_suggested"`
+	TotalLinesAccepted    int                      `json:"total_lines_accepted"`
+	TotalActiveUsers      int                      `json:"total_active_users"`
+	TotalChatAcceptances  int                      `json:"total_chat_acceptances"`
+	TotalChatTurns        int                      `json:"total_chat_turns"`
+	TotalActiveChatUsers  int                      `json:"total_active_chat_users"`
```

Comment:
I'm wondering if any of these `int` fields might benefit from being `int64`, but then I think that that would have to be some crazy Copilot usage to overflow a 32-bit `int`.

So if you want to leave them all as `int` that is fine with me... I just wanted to throw that out there.

Reply:
Valid question. I assumed `int` is taken as a 64 bit int as i `expected` all systems to be 64 bit.

```
The int, uint, and uintptr types are usually 32 bits wide on 32-bit systems and 64 bits wide on 64-bit systems. When you need an integer value you should use int unless you have a specific reason to use a sized or unsigned integer type. 
```

But, obviously this does not have to be true. Lets be on the safe side and just make it int64 as it will most of the time be that anyway!